src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -40,6 +40,21 @@
     end
   end
 
+  describe '#word_break' do
+    it "continuously adds tag after N characters" do
+      expect(word_break("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", 10)).to \
+        eq("XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX<wbr>XXXXXXXXXX")
+    end
+
+    it "adds no tag if string is shorter than N characters" do
+      expect(word_break("XX", 10)).to eq("XX")
```

Comment:
So what I want to say: You should test the boundaries!

And as a suggestion to improve readability: I would change `x...x` to `1234..789` as I don't have to manually count the characters!